src/btree/bt_vrfy.c

```diff
@@ -260,12 +266,18 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
      * itself.
```

Comment:
Can we update line `263` with:
```suggestion
     * Skip the history store explicit call if one of the following conditions is true:
```

Reply:
Done